analysis/study_definition.py

```diff
@@ -1,13 +1,344 @@
-from cohortextractor import StudyDefinition, patients, codelist, codelist_from_csv  # NOQA
+######################################
 
+# This script provides the formal specification of the study data that will be extracted from
+# the OpenSAFELY database.
 
+######################################
+
+
+# IMPORT STATEMENTS ----
+
+# Import code building blocks from cohort extractor package
+from cohortextractor import (
+    StudyDefinition,
+    patients,
+    Measure,
+    codelist,
+)
+
+# Import codelists from codelist.py (which pulls them from the codelist folder)
+from codelists import *
+
+
+## Define study population and variables
 study = StudyDefinition(
-    default_expectations={
-        "date": {"earliest": "1900-01-01", "latest": "today"},
-        "rate": "uniform",
-        "incidence": 0.5,
+
+  # Set index date
+  index_date = "2018-01-01",
+  
+  # Configure the expectations framework
+  default_expectations={
+    "date": {"earliest": "2018-01-01", "latest": "2022-03-31"},
+    "rate": "uniform",
+    "incidence": 0.1,
+  },
+ 
+  # Define the study population
+  # TODO: for TPP studies, do we need to specify TPP registered?
+  population = patients.satisfying(
+      """
+      NOT has_died
+      AND
+      registered
+      AND 
+      (sex = "M" OR sex = "F")
+      AND
+      (age >=18 AND age < 110)
```

Comment:
I wonder should we include children as well?

It may be separate paper but would be good to pull the figures as part of same project. Perhaps start at 5+

Reply:
Sure, easy enough to add